test/unittest/tests/wrappers/mock_session.cpp

```diff
@@ -34,3 +41,18 @@ MockSession::buildTestMockSession()
     // Construct an object that will now own the two pointers passed in.
     return std::shared_ptr<MockSession>(new MockSession(sessionImpl, mockConnection));
 }
+
+int
+handleWiredTigerError(
+  WT_EVENT_HANDLER *handler, WT_SESSION *session, int error, const char *message)
+{
+    ((handler_wrap *)handler)->mock_session->add_callback_message(message);
```

Comment:
This function could simply call `handleWiredTigerMessage` to avoid duplicating code.

Reply:
Done